development/README.md

```diff
@@ -0,0 +1,27 @@
+# Mimir Development
+
+Mimir offers a few docker-compose based development environments for different [deployment modes][deployment-modes], including microservices mode, monolothic mode, and read-write mode.
+
+## Running
+
+Choose one of the sub-folders for whichever Mimir deployment mode you want to run. Then, to build Mimir from source and run it via docker-compose, run:
+
+```bash
+./compose-up.sh
+```
+
+This should give you a running Mimir system with Grafana available at [htttp://localhost:3000](http://localhost:3000), and other Mimir service UIs available via different ports (check `docker ps` for exact ports).
```

Comment:
Possible future improvement: could we auto-generate a list of services and links to them from their Jsonnet definitions?

Reply:
This is an interesting idea. We could maybe use yq to extract the host/port combos for each service from the docker-compose files, write them to a static file, and include the file as a list in the readme using Hugo. Maybe in a followup PR?

Reply:
Yep, definitely a follow-up PR :)